nntrainer/src/nntrainer_logger.cpp

```diff
@@ -88,26 +88,26 @@ void Logger::log(const std::string &message,
   std::stringstream ss;
   switch (loglevel) {
   case NNTRAINER_LOG_INFO:
-    ss << "[NNTRAINER INFO  ";
```

Comment:
when nntrainer is used with different libraries like nnstreamer etc, this might help differentiate nntrainer logs with nnstreamer logs or in tizen etc.

Reply:
Yes. How about NNTR ? it is just to long :(

Reply:
Yes, but for the chance that it will be used for other distros, lets keep it uniform.
NNTR or TRAIN or TRAINER etc.

Reply:
I think NNTRAINER is ok.

Reply:
Add tag done.